tutorials/relay_quick_start.py

```diff
@@ -0,0 +1,141 @@
+"""
```

Comment:
what original tutorial correspond to this one?

Reply:
@masahi https://docs.tvm.ai/tutorials/nnvm_quick_start.html#sphx-glr-tutorials-nnvm-quick-start-py